src/pages/Editor/Classes/styles.tsx

```diff
@@ -14,3 +15,39 @@ export const StyledDateString = styled.span`
   font-weight: 100;
   font-style: italic;
 `;
+
+export const StyledForm = styled.form`
+  margin: 0 5% 0 5%;
+  font-family: 'Poppins';
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+`;
```

Comment:
```suggestion
export const StyledForm = styled(Form)`
  margin: 0 5% 0 5%;
  gap: 1em;
`;
```
if possible, try to not use % for the margin